### PR TITLE
Added reactor-grpc v1.2.4 (#701)

### PR DESCRIPTION
**Pull request by andrewbasterfield**

### CODE REVIEW

File: plugins/community/salesforce-reactive-grpc-java/v1.2.4/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FROM adoptopenjdk/openjdk11:jre-11.0.20_8-debianslim
```
Comment (mfridman) [inline plugins/community/salesforce-reactive-grpc-java/v1.2.4/Dockerfile:1]: A few notes on the Dockerfile

- Ideally, we use one of the base images tracked in https://github.com/bufbuild/plugins/tree/main/.github/docker
- Can the resulting plugin be used in a distroless image
- We typically use `USER nobody`
- Avoid using ARG and hard code the name

**Review by mfridman — COMMENTED** [review]
Thank you for opening a PR, this is more of a drive-by to get the PR consistent with the other dockerized plugins.

I'll bring this up with the team to gauge whether we want to add this as a community plugin.

Is this blocking you, or is this a nice-to-have contribution?

File: plugins/community/salesforce-reactive-grpc-java/v1.2.4/Dockerfile (hunk 1 above)
Comment (andrewbasterfield) [reply plugins/community/salesforce-reactive-grpc-java/v1.2.4/Dockerfile]: Unfortunately the plugin cannot be used in a distroless image as the native executable is a [golang canteen launcher](https://github.com/salesforce/grpc-java-contrib/tree/master/canteen) which has a jar concatenated on the end of it. A JVM is still required in the image. Other issues resolved.

**Comment by andrewbasterfield** [comment]
> Thank you for opening a PR, this is more of a drive-by to get the PR consistent with the other dockerized plugins.
> 
> I'll bring this up with the team to gauge whether we want to add this as a community plugin.
> 
> Is this blocking you, or is this a nice-to-have contribution?

This is a nice-to-have for us

**Comment by pkwarren** [comment]
Before committing to long term maintenance of this plugin, we're interested to see if https://github.com/salesforce/reactive-grpc/issues/312 completes first. It would be good to know it is under active development.

**Comment by pkwarren** [comment]
Thank you for the contribution. We're going to close out this PR for now, as it appears this plugin may not be currently under active development. We're happy to reconsider in the future.

**Pull request closed without merging**